src/main/scala/com/bedrockstreaming/data/sparktest/CustomPrettifier.scala

```diff
@@ -0,0 +1,56 @@
+package com.bedrockstreaming.data.sparktest
+
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.functions.col
+import org.scalactic.Prettifier
+import java.lang.reflect.Method
+
+import CustomerPrettifier.prettyDataFrame
+
+trait CustomPrettifier {
+
+  implicit val customPrettifier: Prettifier = {
+    case df: DataFrame => prettyDataFrame(df)
+    case anythingElse  => Prettifier.default(anythingElse)
+  }
+}
+
+object CustomerPrettifier {
+
+  private[sparktest] def prettyDataFrame(df: DataFrame): String = {
+    val schemaTitle =
+      """
+        |************************************
+        |************** SCHEMA **************
+        |************************************
+        |""".stripMargin
+    val dataTitle =
+      """
+        |************************************
+        |*************** DATA ***************
+        |************************************
+        |""".stripMargin
+
+    val byColNameDF = df.select(df.columns.sorted.map(col): _*)
+    val stringSchema = byColNameDF.schema.treeString
+    val orderedDF = byColNameDF.orderBy(byColNameDF.columns.map(col): _*)
+    val stringData = showString(orderedDF)
+
+    Seq(schemaTitle, stringSchema, dataTitle, stringData).mkString("\n", "\n", "\n")
```

Comment:
Not totally sure about that, but will it work on Windows?

Reply:
Good point, to be sure we could add `\r\n`